vector/src/main/res/layout/fragment_login.xml

```diff
@@ -87,6 +87,7 @@
                     android:layout_marginTop="8dp"
                     android:background="?attr/selectableItemBackground"
                     android:scaleType="center"
+                    android:contentDescription="@string/a11y_show_password"
```

Comment:
for attributes that are changed in code we usually put a tools: instead of android:

Reply:
I do it